lib/SILGen/SILGenPoly.cpp

```diff
@@ -84,7 +84,11 @@
 
 #include "SILGen.h"
 #include "SILGenFunction.h"
+// SWIFT_ENABLE_TENSORFLOW
+#include "SILGenFunctionBuilder.h"
```

Comment:
It's possible to avoid introducing these dependencies by moving the "thunk function creation" logic to SILGenThunk.cpp.

This is done for reabstraction thunks:
* Thunk `SILFunction ` creation is done in SILGenThunk.cpp in `SILGenModule::getOrCreateReabstractionThunk`. SILGenThunk.cpp has the appropriate headers.
* Thunk body creation is done here in SILGenPoly.cpp.

Consider refactoring at some point.